src/BLDC_sm.c

```diff
@@ -397,9 +397,9 @@ void BL_State_Ctrl(void)
     }
     else if( BL_CLS_LOOP == bl_opstate )
     {
-#define CL_FAULT_CNTR 2000
-      const uint8_t FAULT_INCR = 20;
-      const uint8_t FAULT_DECR = 1;
+#define CL_FAULT_CNTR 2000u
```

Comment:
Should we change line 400  #define to "static const uint16_t CL_FAULT_CNTR= 2000u;"  ... for consistency with other static consts?

Reply:
No, it breaks the initialization of the variable that it's used to initialize